modules/broker-coordination/src/main/java/org/wso2/broker/coordination/HaStrategy.java

```diff
@@ -42,6 +42,16 @@
      */
     void stop();
 
+    /**
+     * Method to pause the HA strategy.
+     */
+    void pause();
```

Comment:
It seems these methods are only used in tests. I'm not sure if we should introduce methods in an interface which are only used in tests. WDYT?